### DISCUSSION TITLE
Configure `Railway app` "This page is taking too long to load."

### DISCUSSION THREAD

**Discussion by kareem-g** [community/community#35473] [Other Feature Feedback, Questions, & Ideas] [answered]
hi, I'm trying to authenticate my GitHub to the railway App and I get this error!

![image](https://user-images.githubusercontent.com/66156712/194634609-d4d1a543-8f1b-4681-ad2d-5695cc879db7.png)

**Comment by mr-palindrome** [comment, marked as answer]
I think I found the solution. Try leaving those organisation where you are collaborator!
It worked for me!

**Reply by kareem-g** [reply]
oh, i didn't notice the notification, yeah I left the CS50 organization, and it worked!! anyway thanks <3

**Comment by BhawickJain** [comment]
Hey folks, I can also confirm that @mr-palindrome solution worked! Thanks for this. The organisation I was part of was too large, which caused the failure.

**Comment by OrnellaTchinda** [comment]
Thanks @mr-palindrome Was stuck for 2 days trying to use railway

**Comment by Et7f3** [comment]
This solution is quick fix but It might not be a suitable long term solution if we can't use GitHub org. I would opt to re-open it

**Comment by fahmifadillah** [comment]
verified account github to railway but unverified account
